src/androidTest/java/com/mixpanel/android/mpmetrics/FCMMessagingServiceTest.java

```diff
@@ -91,6 +94,14 @@ private void verifyBasicNotification(Notification.Builder builder, long now) {
         verify(builder).setLargeIcon(any(Bitmap.class));
     }
 
+    private void verifyLargeIcon(Notification.Builder builder) {
+        verify(builder).setLargeIcon(any(Bitmap.class));
+    }
+
+    private void verifyExpandableImage(Notification.Builder builder) {
```

Comment:
I think this is basically just a duplicate of `verifyExpandableNotification`, not sure it's providing any value.